# Increase timeouts to prevent tornado from killing jupyter on compute nodes
diff -Naur tornado.orig/tornado-6.1/tornado/httpclient.py tornado/tornado-6.1/tornado/httpclient.py
--- tornado.orig/tornado-6.1/tornado/httpclient.py	2020-10-30 21:17:45.000000000 +0100
+++ tornado/tornado-6.1/tornado/httpclient.py	2022-05-01 22:01:50.923741948 +0200
@@ -345,8 +345,8 @@
     # Merged with the values on the request object by AsyncHTTPClient
     # implementations.
     _DEFAULTS = dict(
-        connect_timeout=20.0,
-        request_timeout=20.0,
+        connect_timeout=60.0,
+        request_timeout=1200.0,
         follow_redirects=True,
         max_redirects=5,
         decompress_response=True,

